packages/components/src/__actions__/Button/v3/_docs/Button--api-specification.mdx

```diff
@@ -0,0 +1,160 @@
+import { Canvas, Meta, Controls } from "@storybook/blocks"
+import { ResourceLinks, KAIOInstallation } from "~storybook/components"
+import * as specStories from "./Button.spec.stories"
+import * as exampleStories from "./Button.stories"
+
+<Meta title="Actions/Button/Button (v3)/API Specification" />
+
+  # Button API Specification (v3)
+
+  Updated Nov 11, 2024
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/__actions__/Button/v3"
+  figma="https://www.figma.com/design/eZKEE5kXbEMY3lx84oz8iN/%F0%9F%92%9C-Heart-UI-Kit?m=auto&t=a6VcoKBJx6snn0ut-1"
+  ariaComponentPage="https://react-spectrum.adobe.com/react-aria/Button.html"
+  designGuidelines="/?path=/docs/actions-button-v3-usage-guidelines--docs"
+/>
+
+<KAIOInstallation exportNames={["Button" ]} family="actions" version="3" />
+
+## Overview
+
+`Button` allows users to perform an action. If the component needs to navigate somewhere and can be opened in a new tab, use a [link instead](#buttons-and-links).
+
+<Canvas of={exampleStories.Playground} />
+
+## API
+
+This is built on top [React Aria's Button component](https://react-spectrum.adobe.com/react-aria/Button.html#props) and seeks to provide a flexible API with convenient abstractions that handle variants, sizes, pending states and icons.
+
+The following table showcases the essential props that enable the core functionality of `Button`.
+
+<Controls of={exampleStories.Playground} className="mb-64" include={["children", "className",  "size", "variant", "onPress", "icon", "iconPosition", "isFullWidth", "isDisabled", "isPending", "isPendingLabelHidden",  "pendingLabel"]} />
+
+### `onPress` vs `onClick`
+
+One key change to the `Button`&apos;s API is the shift from `onClick` to `onPress`. This will not require a change to the implementation of `onClick` logic, ie:
+
+```tsx
+<Button label="Submit" onClick={e => sumbit(e)}/>
+```
+
+Can safely be replaced with the following:
+
+```tsx
+<Button onPress={e => submit(e)}>
+  Submit
+</Button>
+```
+
+React Aria&apos;s implementation of `onPress` provides better support for consistent touch events across device types, such mobile, desktop and tablet. You can read more about the development and reason behind this pattern [here](https://react-spectrum.adobe.com/blog/building-a-button-part-1.html#touch-interactions).
```

Comment:
Reads better if we move this above the code snippets.
Why then How (as opposed to the opposite)